src/Raven.Client/Documents/Operations/Replication/UpdatePullReplicationAsSinkOperation.cs

```diff
@@ -12,10 +12,23 @@
 
 namespace Raven.Client.Documents.Operations.Replication
 {
+    /// <summary>
+    /// Operation to update the configuration of a pull replication task as a sink.
+    /// Pull replication as a sink allows a database to pull data from a source (hub) database in another cluster or server.
+    /// </summary>
     public sealed class UpdatePullReplicationAsSinkOperation : IMaintenanceOperation<ModifyOngoingTaskResult>
     {
         private readonly PullReplicationAsSink _pullReplication;
 
+        /// <inheritdoc cref="UpdatePullReplicationAsSinkOperation"/>
+        /// <param name="pullReplication">
+        /// The <see cref="PullReplicationAsSink"/> object containing the updated configuration for the pull replication sink task.
```

Comment:
need to document `PullReplicationAsSink` class

Reply:
done